modules/db/src/test/java/org/mule/module/db/integration/template/TemplateQueryConfigTestCase.java

```diff
@@ -132,6 +132,23 @@ public void usesDynamicQueryWithPlaceholder() throws Exception
     }
 
     @Test
+    public void usesNullDefaultParamValue() throws Exception
+    {
+        Object queryTemplateBean = muleContext.getRegistry().get("testNullParamsQuery");
+
+        QueryTemplate queryTemplate = (QueryTemplate) queryTemplateBean;
+
+        assertFalse(queryTemplate.isDynamic());
+        assertEquals("SELECT * FROM PLANET WHERE POSITION = ?", queryTemplate.getSqlText());
+        assertEquals(1, queryTemplate.getInputParams().size());
+        InputQueryParam param1 = queryTemplate.getInputParams().get(0);
+        assertEquals(UnknownDbType.getInstance(), param1.getType());
+        assertEquals("position", param1.getName());
+        assertEquals(null, param1.getValue());
```

Comment:
Should I compare with null value or with "null" literal Value?
What is the expected behavior?
Look at checkNullValue method.